gnosis/safe/api/base_api.py

```diff
@@ -36,6 +37,7 @@ def __init__(
         if not self.base_url:
             raise EthereumNetworkNotSupported(network)
         self.http_session = requests.Session()
```

Comment:
Maybe you forgot to enable here the pool connection for this requests?

Reply:
Not really required, but adding it